### PR TITLE
feat: add notification about available new version

### LINKED ISSUE #779: Update outdated version warning in output
**Is your feature request related to a problem? Please describe.**

There is no good way to alert users to new updates in the CLI tool.

**Describe the solution you'd like**

I'd like part of the CLI output to check and show the current version versus the running version. If the current version is different, to show this output... something like

```bash
A new version of Redocly CLI (1.0.0-beta.105) is available.
Update now: npx @redocly/cli@latest upgrade
Changelog: {TODO}
```

Optimal would be an integrated changelog information and show every change between the version installed and the current version.

**Describe alternatives you've considered**

None yet.

**Additional context**

Consider also the JSON output format so that it can be parsed and programmatically displayed in Workflows.

### PR DESCRIPTION
**Pull request by AntonKozachuk**
## What/Why/How?
Add notification to `redocly-cli` about the new version is available.

## Reference
Closes #779.

## Testing
Tested locally.

## Screenshots (optional)
image

## Check yourself

- [x] Code is linted
- [ ] Tested with redoc/reference-docs/workflows
- [ ] All new/updated code is covered with tests

## Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### CODE REVIEW

**Comment by AntonKozachuk** [comment]
@adamaltman @RomanHotsiy do we want the notification to look consistent with the one from `redoc-cli` or do it the way it was described in the issue?

image

or 

image

**Comment by adamaltman** [comment]
I think the second screenshot you posted is better.

**Comment by tatomyr** [comment]
To me the second also looks better, but without the backticks around the version.

File: packages/cli/src/utils.ts [hunk 1]
```diff
@@ -323,3 +324,15 @@ export function isSubdir(parent: string, dir: string): boolean {
...
+
+export function notifyUpdateCliVersion() {
+  const pkg = require('../package.json');
+  const notifier = updateNotifier({
+    pkg,
+    updateCheckInterval: 0,
+    shouldNotifyInNpmScript: true,
+  });
+  notifier.notify({
+    message: 'Run `{updateCommand}` to update.\nChangelog: https://redocly.com/docs/cli/changelog',
```
Comment (tatomyr) [inline packages/cli/src/utils.ts:336]: Please update the message.

**Review by tatomyr — APPROVED** [review]

**Review by RomanHotsiy — APPROVED** [review]

**Pull request merged**